profile/README.md

```diff
@@ -1,3 +1,13 @@
 # PaperMC
 
 PaperMC is a Minecraft Software organization focusing on improving the Minecraft ecosystem with faster and more secure software.
+
+**Quickstart:**
+- Learn more on [PaperMC.io](https://papermc.io) or [get started](https://docs.papermc.io/paper/getting-started) now!
+- You can download Paper [here](https://papermc.io/downloads)
```

Comment:
```suggestion
- Learn more and find downloads on [papermc.io](https://papermc.io)
```
The get started guide isn't org related

Reply:
Are you sure that you dont want to link directly to the downloads page?

Reply:
Yes

Reply:
Okay 😃